src/Uno.UWP/Devices/Geolocation/Geolocator.Android.cs

```diff
@@ -310,7 +310,12 @@ private async void InitializeIfPermissionIsGranted()
 		RestartUpdates();
 	}
 
-	partial void StopPositionChanged() => _positionChangedSubscriptions.TryRemove(this, out var _);
+	partial void StopPositionChanged()
+	{
+		_positionChangedSubscriptions.TryRemove(this, out _);
+		RemoveUpdates();
+		BroadcastStatusChanged(PositionStatus.Disabled);
```

Comment:
@jeromelaban I think https://github.com/unoplatform/uno/pull/12091#pullrequestreview-1404924397 needs to be addressed first. Making this comment to temporary block auto merge.

Reply:
Oh actually I can no longer make any sense of my comment 😄 
Resolving this for now. I might have read something wrong.